src/frontend/src/lib/components/hero/HeroSignIn.svelte

```diff
@@ -19,5 +18,7 @@
 </div>
 
 <ButtonAuthenticate on:click={async () => await signIn({})}>
-	<Html text={$i18n.auth.text.authenticate} />
+	{$i18n.core.text.open}
```

Comment:
We do not need a style `gap` somewhere if we remove the `nbsp;`?

Reply:
I put the `|` character in a span with margins, and it seems to be working well. But it is the same, as you said: i can put them inside something with a `gap` instead of this. What do you prefer? No preference from me

Reply:
Gotcha. All good.